diff --git a/ppocr/data/imaug/random_crop_data.py b/ppocr/data/imaug/random_crop_data.py
index c7ac5f2..d7ccb89 100644
--- a/ppocr/data/imaug/random_crop_data.py
+++ b/ppocr/data/imaug/random_crop_data.py
@@ -74,7 +74,7 @@ def region_wise_random_select(regions, max_size):
     selected_values = []
     for index in selected_index:
         axis = regions[index]
-        xx = int(np.random.choice(axis, size=1))
+        xx = int(np.random.choice(axis))
         selected_values.append(xx)
     xmin = min(selected_values)
     xmax = max(selected_values)
